### PR TITLE
chore(ci): delivery to new rpm and apt repositories

### PR DESCRIPTION
**Pull request by kduret**
## Description

chore(ci): delivery to new rpm and apt repositories

**Fixes** MON-14127

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x (master)

### CODE REVIEW

File: .github/workflows/get-version.yml [hunk 1]
```diff
@@ -98,7 +98,7 @@ jobs:
               STABILITY="stable"
               ;;
             *)
-              STABILITY="canary"
+              STABILITY="unstable"
```
Comment (kduret) [inline .github/workflows/get-version.yml:101]:
```suggestion
              STABILITY="canary"
```

**Review by tuntoja — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

File: .github/actions/backend-dependencies/action.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ runs:
       shell: bash
 
     - name: Cache vendor directory
-      uses: actions/cache@69d9d449aced6a2ede0bc19182fadc3a0a42d2b0
+      uses: actions/cache@v3
```
Comment (paloth) [inline .github/actions/backend-dependencies/action.yml:27]: Isn't it better to fix the commit sha than the tag ?

Comment (kduret) [reply .github/actions/backend-dependencies/action.yml:27]: I think we'll may need to use tag instead : https://github.com/actions/checkout/releases/tag/v3.5.0

Comment (kduret) [reply .github/actions/backend-dependencies/action.yml:27]: I've restored v3 for consistency with develop branch

**Pull request merged**